battleship/SaveLoad.java

```diff
@@ -0,0 +1,88 @@
+package battleship;
+
+import java.io.*;
+import java.util.ArrayList;
+
+import battleship.Attack.*;
+import battleship.Model.*;
+
+
+
+
+public class SaveLoad {
+
+    AI ai = new AI();
+    
+    Board ourBoard = new Board(Constants.boardSize);
+	Board enemyBoard = new Board(Constants.boardSize);
+    Board board = new Board(Constants.boardSize);
+
+    public void Save(int enemyMisses, int enemyHits, int ourHits, int ourMisses, ArrayList<Coordinate> pastShots){
+        try{
+            FileOutputStream saveFile = new FileOutputStream("SaveFile.sav");
```

Comment:
not sure about using sav file lmao, let's just use .txt
and in that case we can use bufferedwriter/filewriter (?)